### PR TITLE
DRILL-6696: IOBE in Operator Metric Registry

### PR DESCRIPTION
**Pull request by vdiravka**

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/ops/OperatorMetricRegistry.java [hunk 1]
```diff
@@ -61,14 +66,11 @@
...
     // Currently registers a metric def that has enum constants
-    final MetricDef[] enumConstants = metricDef.getEnumConstants();
-    if (enumConstants != null) {
-      final String[] names = new String[enumConstants.length];
-      for (int i = 0; i < enumConstants.length; i++) {
-        names[i] = enumConstants[i].name();
-      }
-      OPERATOR_METRICS[operatorType] = names;
-    }
+    String[] names = Arrays.stream(metricDef.getEnumConstants())
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/main/java/org/apache/drill/exec/ops/OperatorMetricRegistry.java]: Do you know the case when enum constants are null?

File: exec/java-exec/src/test/java/org/apache/drill/TestOperatorMetrics.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+@Category(OperatorTest.class)
+public class TestOperatorMetrics extends BaseTestQuery {
+
+  @Test
+  public void testMetricNames() {
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/test/java/org/apache/drill/TestOperatorMetrics.java]: Maybe split into two tests: negative and positive?

**Comment by arina-ielchiieva** [comment]
Please rename the commit message to reflect the actual changes.

File: exec/java-exec/src/main/java/org/apache/drill/exec/ops/OperatorMetricRegistry.java (hunk 1 above)
Comment (vdiravka) [reply exec/java-exec/src/main/java/org/apache/drill/exec/ops/OperatorMetricRegistry.java]: My bad, not right `null` check. EnumConstants array should be checked.

File: exec/java-exec/src/test/java/org/apache/drill/TestOperatorMetrics.java (hunk 2 above)
Comment (vdiravka) [reply exec/java-exec/src/test/java/org/apache/drill/TestOperatorMetrics.java]: It makes sense. Done

**Comment by arina-ielchiieva** [comment]
+1, LGTM.

**Pull request merged**